wsd/COOLWSD.cpp

```diff
@@ -4594,7 +4594,7 @@ class ClientRequestDispatcher final : public SimpleSocketHandler
         {
             // Do things in the right thread.
             LOG_TRC_S("Move media request " << tag << " to docbroker thread");
-            docBroker->handleMediaRequest(socket, tag);
+            docBroker->handleMediaRequest(request, socket, tag);
```

Comment:
Let's pass in the `range`. No reason to depend on `Poco::Net::HTTPRequest`  to read a header.